<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRouterClass</class>
 <widget class="QWidget" name="QRouterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>958</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QRouter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_send1">
       <property name="text">
        <string>send event to page1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_send_cur">
       <property name="text">
        <string>send event to page current</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_send_all">
       <property name="text">
        <string>send event to all page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="send_by_post">
       <property name="text">
        <string>post method</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>stack widget:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">QStackedWidget{border:1px solid red;}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>widget:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget{
border:1px solid red;
}</string>
     </property>
     <layout class="QVBoxLayout" name="container_layout"/>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="log_view"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qrouter.qrc"/>
 </resources>
 <connections/>
</ui>
